README.txt

```diff
@@ -1,21 +1,24 @@
-Methodocracy
+Methodocracy (noun):
+A cultural-political ideology which proposes that optimal decision-making, from the individual to national level, is necessarily based upon the quantity and quality of data, interpretation, and conclusions gathered through tested and reliable methodologies.
 
-"The Super-Power of Problem-Solving"
 
```

Comment:
At this stage, you're trying to recruit scientists and tech professionals yes? I don't think a tagline this common sounding will get their attention. In fact, I fear it may be so generic that they would not take it seriously. There have too many movies out about super  heroes and super powers lately.

I think something more succinct could be "Proliferating Science".